contrib/build_sdk.sh

```diff
@@ -338,7 +340,7 @@ openssl_pkg() {
     fi
     local loc_make_opts=$make_opts
 
-    if [ $incremental -eq 1 ] && [ -e $status_dir/$name.success ]; then
+    if [ $incremental -eq 1 ] && [ -e $status_dir/$name.success ] && [ `cat $status_dir/$name.success` = $openssl_md5 ]; then
```

Comment:
There's a little backwards compatibility issue with this & similar changes:
for systems relying on $name.success in the old ways (content = `0`), this will fail.